src/executeCommandLine/executeCommandLine.ts

```diff
@@ -871,8 +872,8 @@ function performBuild(
         reportDiagnostic,
         createBuilderStatusReporter(sys, shouldBePretty(sys, buildOptions)),
         createReportErrorSummary(sys, buildOptions),
+        jsDocParsingMode,
     );
-    buildHost.skipNonSemanticJSDocParsing = skipNonSemanticJSDocParsing;

```

Comment:
Through the process of ensuring I plumbed this option properly everywhere, this (and other assignments) turned into a parameter on the create function. I'm not sure which is better, but I feel like providing it up front is clearer and matches what we do for `createIncrementalCompilerHost`, `createCompilerHostWorker`, etc, and is likely more discoverable than having to patch the host afterwards. But, I'm not picky here.

Reply:
Okay, per PR feedback, these are back to being assignments.